src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -1,736 +1,769 @@
-                {
-                    _applicationSettings.Remove(change.Key);
-                }
-                else if (change.ChangeType == KeyValueChangeType.Modified)
-                {
-                    _applicationSettings[change.Key] = change.Current;
-                }
-
-                // Invalidate the cached Key Vault secret (if any) for this ConfigurationSetting
-                foreach (IKeyValueAdapter adapter in _options.Adapters)
-                {
-                    adapter.InvalidateCache(change.Current);
-                }
-            }
-        }
-
-        private Task CallWithRequestTracing(Func<Task> clientCall)
-        {
-            var requestType = _isInitialLoadComplete ? RequestType.Watch : RequestType.Startup;
-            return TracingUtils.CallWithRequestTracing(_requestTracingEnabled, requestType, _requestTracingOptions, clientCall);
-        }
-
-        private void SetRequestTracingOptions()
-        {
-            _requestTracingOptions = new RequestTracingOptions
-            {
-                HostType = TracingUtils.GetHostType(),
-                IsDevEnvironment = TracingUtils.IsDevEnvironment(),
-                IsKeyVaultConfigured = _options.IsKeyVaultConfigured,
-                IsKeyVaultRefreshConfigured = _options.IsKeyVaultRefreshConfigured,
-                IsOfflineCacheConfigured = _options.IsOfflineCacheConfigured
-            };
-        }
-
-        private DateTimeOffset AddRandomDelay(DateTimeOffset dt, TimeSpan maxDelay)
-        {
-            long randomTicks = (long)(maxDelay.Ticks * RandomGenerator.NextDouble());
-            return dt.AddTicks(randomTicks);
-        }
-
-        private bool IsAuthenticationError(Exception ex)
-        {
-            if (ex is RequestFailedException rfe)
-            {
-                return rfe.Status == (int)HttpStatusCode.Unauthorized || rfe.Status == (int)HttpStatusCode.Forbidden;
-            }
-
-            if (ex is AggregateException ae)
-            {
-                return ae.InnerExceptions?.Any(inner => IsAuthenticationError(inner)) ?? false;
-            }
-
-            return false;
-        }
-
-        private void UpdateCacheExpirationTime(KeyValueWatcher changeWatcher, bool success)
-        {
-            TimeSpan cacheExpirationTime;
-
-            if (success)
-            {
-                changeWatcher.RefreshAttempts = 0;
-                cacheExpirationTime = changeWatcher.CacheExpirationInterval;
-            }
-            else
-            {
-                if (changeWatcher.RefreshAttempts < int.MaxValue)
-                {
-                    changeWatcher.RefreshAttempts++;
-                }
-
-                cacheExpirationTime = changeWatcher.CacheExpirationInterval.CalculateBackoffTime(changeWatcher.RefreshAttempts);
-            }
-
-            changeWatcher.CacheExpires = DateTimeOffset.UtcNow.Add(cacheExpirationTime);
-        }
-    }
-}
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
this whole file is showing up as a change.

Reply:
That's strange. CodeFlow shows only the new changes.